sessionctx/variable/varsutil.go

```diff
@@ -625,6 +625,26 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			return "", nil
 		}
 		return value, ErrWrongValueForVar.GenWithStackByArgs(name, value)
+	case TiDBIsolationReadEngines:
```

Comment:
please add some UT for this `switch` branch

Reply:
Done.